Application/EdFi.Ods.Api/Container/Modules/PostgresSpecificModule.cs

```diff
@@ -28,10 +28,6 @@ public override void ApplyConfigurationSpecificRegistrations(ContainerBuilder bu
                 .As<IParameterListSetter>()
                 .SingleInstance();
 
-            builder.RegisterType<PostgresAuthorizationSegmentSqlProvider>()
```

Comment:
The next using is no longer needed:
`using EdFi.Ods.Api.Security.Authorization;`

Reply:
Done.